index.html

```diff
@@ -50,16 +48,11 @@ <h2>Things that Betina dislikes</h2>
         <li><strong>Being dirty:</strong> Betina may not like getting dirty and may not enjoy playing in mud or dirty environments.</li>
         <li><strong>Strict discipline:</strong> Betina may not respond well to strict discipline or harsh training methods. She may be better trained with positive reinforcement and rewards-based methods.</li>
         </ol>
-
         <hr>
-
         <div>
         <button class="btn">Click if you liked<img src="imagenes/thumbs-up-solid.svg" alt="ThumbsUp" class="thumbs"></button>
         </div>
-
-
-
 </main>
```

Comment:
- [ ] Please, make sure you add the 'why' you've suggested any changes, this creates a conviction for the reviewee 😀.
your first reviewer about header has a why, but the second one with the commit messages need some clean up. 😉